l10n_co_geo/data/res_bank.xml

```diff
@@ -0,0 +1,864 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this an imperative or an optional recommendation? We have a module `l10n_co_terceros` which is loading additional data, also on banks. Do you mean it would be wise to split out the loading of banks out of this module? Let's say into a `l10n_co_bank_data`? I can see an advantage of splitting things into smaller units to benefit more focused discussion and collaboration. Asking for clarification...